src/pages/payment-methods/index.tsx

```diff
@@ -120,7 +123,11 @@ const MobileWrapper = styled.div`
         display: block;
     }
 `
-const DisplayAccordion = (locale) => {
+type PaymentMethodsProps = {
+    locale: object
+    pd?: object
```

Comment:
Can we add the fields from `pd` so we don't need to do this `pd['is_fiat_onramp']` below.